mkdocs.yml

```diff
@@ -7,6 +7,7 @@ extra_css:
   - stylesheets/extra.css
 site_dir: site
 docs_dir: docs
+copyright: The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For a list of TLF trademarks, see <a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademark Usage</a>.
```

Comment:
thank you!!! do we need to do the same with README or its OK there?

Reply:
It's ok on the website.